### ISSUE TITLE
CIRCexplorer2  support   hisat2

### ISSUE THREAD

**Issue by weizhiting** [YangLab/CIRCexplorer2#16]
note that TopHat has entered a low maintenance, low support stage ,  so does  the circexplorer2 support hisat2 ?

**Comment by kepbod** [comment]
Because currently hisat2 did not support fusion junction searching (https://github.com/infphilo/hisat2/issues/8), so we could not use it to find circRNAs. You could use other alignment tools instead if you do not want to use tophat2.

**Comment by weizhiting** [comment]
Thanks for your reply! one more questions,  whether the 13th column  of the circularRNA_known.txt  which generate by CIRCexplorer2 annotate module  is the expression of the circRNA?  if not ,how can i get the expression?

**Comment by xingma** [comment]
The 13th column represents the number of back-spliced junction reads. It represents the expression level of circRNAs in one sample. However, if you want to compare circRNAs among multiple samples. RPM(number of back-spliced reads per million raw reads) is a better choice.

**Comment by weizhiting** [comment]
got it! But when i use CIRCexplorer2 to analysis my sample( 12G/sample, 3sample), the expression of my circRNA is very low.  The total counts of all the back-spliced junction reads is about 6000 and the highest is 60reads.   Is that normal  or i did something wrong?  I use STAR to align and the parse and annotate module!  my sample is come from rat and I download gtf  from ensembl!

**Comment by kepbod** [comment]
The counts and expression of circRNAs are related with your samples. It is out of what CIRCexplorer2 could do.

**Comment by meghahs** [comment]
Dear kepbod,

I was trying CIRCexplorer2 for the arabidopsis genome.
But I am getting zero hits once I annotate it using the refFlat file.

I used the below codes to obtain the refFlat file
gtfToGenePred -genePredExt -geneNameAsName2 genes.gtf refFlat.tmp.txt
paste <(cut -f 12 refFlat.tmp.txt) <(cut -f 1-10 refFlat.tmp.txt) > refFlat.txt
rm refFlat.tmp.txt
gzip refFlat.txt

I have creaded the bed file from STAR aligner and the file output looks something like below
Chr3	14200203	14202727	FUSIONJUNC_0/1	0	+
Chr3	14202659	14203134	FUSIONJUNC_1/2	0	+
Chr2	8692	9269	FUSIONJUNC_2/1	0	+
Chr3	14200993	14202805	FUSIONJUNC_3/1	0	+

I am not understanding where I am going wrong.
Kindly help to troubleshoot the issue

Regards
Megha
